Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs

```diff
@@ -81,13 +81,43 @@ public static void DoBuildUniversal (string outputPath)
             {
                 throw new Exception(error); // ensures exit code != 0.
             }
+#if UNITY_EDITOR
```

Comment:
@robertocaldas what is your recommendation here for Ruslan, that he remove UNITY_EDITOR or wrap the class in an ifdef. If you can please suggest a solution here that would be great.